### PR TITLE
Initial sphinx documentation

### PR DESCRIPTION
**Pull request by momojohobo**
After merge we can notify read the docs to build the documentation from your master.

### CODE REVIEW

**Comment by TheCheapestPixels** [comment]
Please .gitignore ` docs/_templates/` as well.

File: README.md [hunk 1]
```diff
@@ -7,657 +7,79 @@ optimizations compromise it.
...
-games. The central ideas are that
-* state should be separated from the logic working on it
-* state objects should be extended by using composition instead of classical
-  patterns of inheritance, and be extendable and restrictable at runtime
-* logic processes those objects to which it is applicable, determined by the
-  presence of a fitting set of components.
-* logic is applied in a round-robin fashion; In the context of games that likely
-  means "each piece of logic is applied once per frame, in a predetermined
-  order."
+This keeps data and logic seperated, modular, readable and reusable.
```
[the remark this answers is not in the document]
Comment (momojohobo) [reply README.md]: Done, though trying to use less technical terms for the main readme.

File: wecs/README.md [hunk 2]
```diff
@@ -0,0 +1,650 @@
+## What is an ECS?
```
[the remark this answers is not in the document]
Comment (momojohobo) [reply wecs/README.md]: I added a rough machine conversion of the file to `docs/manual/specs.rst`. Sphinx doesn't take markdown out of the box. `docs/gen.sh`  only generates the docstrings in the wecs python modules and generates all the html, anything else needs to be hand-written.

File: README.md [hunk 3]
```diff
@@ -7,657 +7,79 @@ optimizations compromise it.
...
-
-Now when running the main loop, each `System` will now fetch all `Entity`s that
-have sufficient `Component`s to satisfy one or more of its filters, then update
-them. This may involve updating `Component`s that aren't on any of the
-`System`'s filters, and which may even be on any `Entity` in the `World`.
-
+## Documentation
 
-# API
+We're currently working on documentation. You can see it here: https://wecs.readthedocs.io/en/sphinx-start/
```
[the remark this answers is not in the document]
Comment (momojohobo) [reply README.md]: updated url to link to the one readthedocs renders from your master (this link should start working after merge)

**Pull request merged**